### PR TITLE
update 1.2.2

### PR DESCRIPTION
**Pull request by Arishamays1**
## Version Bump to 1.2.2
- updated checksum and version number
- added skip for `python` versions less than 3.7

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/q/qtawesome/QtAwesome-{{ version }}.tar.gz
-  sha256: d37bbeb69ddc591e5ff036b741bda8d1d92133811f1f5a7150021506f70b8e6e
+  sha256: 97b6c66456592195494993d60ac82cc6b85fe49bfd1f12973eb2ad1fee20fa3f
 
 build:
   number: 0
   skip: true  # [ppc64le or s390x]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: Could you please add skipping `py<37`? See https://github.com/spyder-ide/qtawesome/blob/150e7983d4471a68b27b10ebc6ec35f7d314528c/setup.py#L37

Comment (Arishamays1) [reply recipe/meta.yaml:13]: Added. Thanks, Serhii!

**Review by skupr-anaconda — APPROVED** [review]

**Review by pseudoyim — APPROVED** [review]
👍 LGTM

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**